### ISSUE TITLE
Deprecated: implode(): Passing glue string after array is deprecated

### ISSUE THREAD

**Issue by rolesvillesoftware** [soflyy/oxygen-bugs-and-features#946]
**Describe the bug**
Getting the following error when running WordPress on Apache2, PHP7.4.4 in a docker container Ubuntu 18.05

`
Deprecated: implode(): Passing glue string after array is deprecated. Swap the parameters in /var/www/website/wp-content/plugins/oxygen/component-framework/component-init.php on line 4828

`
**Have you tried all the steps at https://oxygenbuilder.com/documentation/troubleshooting/troubleshooting-guide/?**
Yes

**Are you able to replicate the issue on a Sandbox install at https://oxygenbuilder.com/try?**
No because I do not have access to the container running the sandbox. 

**To Reproduce**
1. Create a docker container - Here is my docker file:

```
FROM ubuntu:latest
MAINTAINER Bryan Lenihan<xxxx@xxx.com>
ENV DEBIAN_FRONTEND=noninteractive

#Install Basics
RUN apt-get update
RUN apt-get install -y software-properties-common
RUN add-apt-repository ppa:ondrej/php
RUN add-apt-repository ppa:ondrej/apache2
RUN apt-get update

RUN apt-get install -y curl

##Install Linux tools
RUN apt-get install -y iputils-ping net-tools
RUN apt-get install -y vim

# Install Apache
RUN apt-get install -y apache2

# Install PHP
RUN apt-get -y install php7.4
RUN apt-get -y install libapache2-mod-php7.4
RUN apt-get -y install php7.4-common php7.4-mysql php7.4-xml php7.4-xmlrpc php7.4-curl php7.4-gd php7.4-imagick php7.4-cli php7.4-dev php7.4-imap php7.4-mbstring php7.4-opcache php7.4-soap php7.4-zip php7.4-intl

## Configure the TZData
RUN ln -fs /usr/share/zoneinfo/America/New_York /etc/localtime
RUN dpkg-reconfigure --frontend noninteractive tzdata

RUN apt-get clean

## Update PHP.init enable tags ang quieten logging
RUN sed -i 's/short_open_tag = Off/short_open_tag = On/' /etc/php/7.4/apache2/php.ini
RUN sed -i 's/error_reporting = .*$/error_reporting = E_ERROR | E_WARNING | E_PARSE/' /etc/php/7.4/apache2/php.ini

# Manually set up the apache environment variables
ENV APACHE_LOG_DIR /var/log/apache2
ENV APACHE_LOCK_Dir /var/lock/apache2
ENV APACHE_PID_FILE /var/run/apache2.pid

#Expose Apache
EXPOSE 80
EXPOSE 8080
EXPOSE 443
EXPOSE 3306

#Update the default apache site with the config we created
ADD apache-config.conf /etc/apache2/sites-enabled/000-default.conf

#By default start up apache in the foreground, override with /bin/bash for interactive
CMD /usr/sbin/apache2ctl -D FOREGROUND

```

2. Start the container with the mount to the source directory container WP
3. Go to the URL and you will get the error when the page is built. 

**Expected behavior**
*A clear and concise description of what you expected to happen.*
Should not see the error and see a clean page. 

**What actually happens**
*A clear and concise description of what happens instead of the expected behavior.*
See several PHP errors mixed in with the page. 

**Screenshots**
*If applicable, add screenshots to help explain your problem.*
![image](https://user-images.githubusercontent.com/29575004/78305475-f6dea600-750e-11ea-935a-9aafb0f0ba69.png)

**Desktop (please complete the following information):**
 - Oxygen Version [e.g. 2.2 Alpha]

**Smartphone (please complete the following information):**
Chrome and Safari
 - Oxygen Version: 3.2.1

**Additional context**
*Add any other context about the problem here.*
Works fine with MacOSX PHP 7.4.4 when running in PhpStorm. However, looking at the code, PHPStorm is highlighting/questioning the parameter order as well.

**Comment by rolesvillesoftware** [comment]
Note: Created a new container with PHP version 7.3 and it works great. No longer a show stopper, but we would like to go with PHP 7.4 ASAP as we have other PHP projects.

**Comment by ddon** [comment]
oh, created the same issue in June, after 2 months this is still an issue and I checked in 3.4 beta 1, the same errors filling up log file pretty rapidly.

**Comment by KittenCodes** [comment]
> PHP Error "implode() and join() - Passing glue string after array is deprecated."

Fixed in v3.6 Beta.
